app/forms/author_form.rb

```diff
@@ -1,19 +1,56 @@
 # frozen_string_literal: true
 
-# Form for an author contributor
-class AuthorForm < ContributorForm
+# Form for an author
+class AuthorForm < ApplicationForm
   attribute :first_name, :string
   validates :first_name, presence: true, if: -> { person? && deposit? }
 
   attribute :last_name, :string
   validates :last_name, presence: true, if: -> { person? && deposit? }
 
+  validate :name_must_be_complete, if: :person?
+
   attribute :organization_name, :string
   validates :organization_name, presence: true, if: -> { organization? && deposit? }
 
+  attribute :person_role, :string
+  validates :person_role, presence: true, if: :person?
+
+  attribute :organization_role, :string
+  validates :organization_role, presence: true, unless: :person?
+
   attribute :role_type, :string
 
-  def organization?
-    role_type == 'organization'
+  attribute :with_orcid, :boolean, default: false
+  attribute :orcid, :string, default: nil
+  validates :orcid, format: { with: /\A\d{4}-\d{4}-\d{4}-\d{3}[0-9X]\z/ },
+                    allow_blank: true,
+                    if: :person?
```

Comment:
This is the bulk of the work in this PR: collapsing the contributor form into the author form. My reasoning is that 1) it makes working with the author form (e.g. mapping it) easier because it's self-contained, and 2) the contributor form isn't used yet so the boundary between these two forms is fuzzy. If we can defer doing something until when we know more or know better, I'd advocate for taking that just-in-time vs. just-in-case approach.